model/Core/Vocabularies/RelationshipType.md

```diff
@@ -66,6 +66,16 @@ Build Profile specific RelationshipType descriptions can be found [here](https:/
 - buildInvokedBy: Agent that invoked the build
 - buildOnBehalfOf: Actor for which buildInvokedBy is acting on behalf of
 - buildHostOf: Element in which the build instance runs on
+- advisory: (Security) Used to associate a security vulnerability with an element
+- coordinatedBy: (Security) TODOdescription
+- exploitCreatedBy: (Security) TODOdescription
+- fixedBy: (Security) Designates a vulnerability has been fixed by an element, actor or entity
+- foundBy: (Security) Designates an actor or entity was the original discoverer of a security exploit
+- hasResponsibleParty: (Security) TODOdescription
+- issuedBy: (Security) Designates an actor or entity that issued an identity (i.e. CVE ID) to the vulnerability originating the relationship
+- publishedBy: (Security) Designates the actor or entity that made a vulnerability record available for public use or reference
+- reportedBy: (Security) Designates the actor or entity that first reported a vulnerability to the project, vendor, or tracking database for formal identification 
+- suppliedBy: (Security) TODOdescription
```

Comment:
We had decided to use Supplier as a property rather than a relationship, and this is likely to add confusion again.   Can a different term be used?

Reply:
Yes, we updated this and I pushed the changes.